docs/components/table.md

```diff
@@ -85,6 +85,76 @@ Lessen the table cell padding
   isSmall />
 ```
 
+### Sorting
+There are two props used to make the table sortable; ```sortOrder```, which is either 'ascending' or 'descending' and ```sortKey```, which tells the table which column is currently being sorted. If a sortKey exists, then clicking the
+table header will emit an Event called ```sort``` which must be handled by the parent component to implement the
+actual sorting logic. A basic implementation of ```sort``` called ```defaultSorter``` is included in KTable and can be imported separately and used with a helper function in the parent.
```

Comment:
this is confusing to me just coming in to this. `Sort` is the handler? and `sortKey` is the key in which the handler sorts the list by? I feel like `sort` and `defaultSorter` should be renamed to be more clear. The word _sort_ feels like a boolean to me

Reply:
maybe `@sorter`? then `defaultSorter` would be more obvious what it's used for.

Reply:
actually, @sorter wouldn't make much sense, because it's an action. `@sort` is read as "on sort". If it was a prop then it would make more sense to interpret as a boolean

Reply:
Im not necessarily saying its wrong, Im just saying looking at this for the first time its confusing to me to interpret what Im supposed to do/what things mean

Reply:
`sort` is the name of the event action unless you have a better name for it

Reply:
Bootstrap uses [sort-compare](https://bootstrap-vue.js.org/docs/components/table/#custom-sort-compare-routine), Im not sure of a better name I just feel sort isn't clear. Again I understand it _now_ but at first read it was not clear. Especially when the function is named "sorter". Maybe `sort-function`, `sort-handler`, `sort-comparer`

Reply:
`sort-compare` is the function / prop name in that example, and `sort` in KTable is the event name. it makes sense to me at least for the action to be `sort` and the function to act on it be the `sorter`